test/Debugger/JsDiagGetStackProperties.js.dbg.baseline

```diff
@@ -3,6 +3,7 @@
     "this": "Object {...}",
     "{exception}": "number 1",
     "locals": {
+      "console": "Object {...}",

```

Comment:
Are you adding this change to jshost too?  These tests will fail with jshost otherwise.